app/mailers/form_submission_mailer.rb

```diff
@@ -1,12 +1,17 @@
 class FormSubmissionMailer < GovukNotifyRails::Mailer
-  def email_completed_form(title:, text_input:, reference:, timestamp:, submission_email:)
+  def email_completed_form(title:, text_input:, reference:, preview_mode:, timestamp:, submission_email:)
     set_template(Settings.govuk_notify.form_submission_email_template_id)
 
     set_personalisation(
       title:,
       text_input:,
       submission_time: timestamp.strftime("%H:%M:%S"),
       submission_date: timestamp.strftime("%-d %B %Y"),
+      # GOV.UK Notify's templates have conditionals, but only positive
+      # conditionals, so to simulate negative conditionals we add two boolean
+      # flags; but they must always have opposite values!
+      test: preview_mode ? "yes" : "no",
+      not_test: preview_mode ? "no" : "yes",
```

Comment:
To make sure that they always have opposite values, you could make a method 

```
def make_boolean_readable(boolean)
  boolean ? "yes" : "no"
end
```

and then do 
```suggestion
      test: make_boolean_readable(preview_mode),
      not_test: make_boolean_readable(!preview_mode),
```